tasks/install_jtdx.yml

```diff
@@ -188,28 +188,69 @@
-      dest: "/home/{{ ham_user }}/hamradio/{{ jtdx_installer }}"
+  # - name: Determine correct JTDX version for Xubuntu/Ubuntu
+  #   set_fact: jtdx_installer="jtdx-2.2.0-rc155_2004_amd64.deb"
+  #   when: ansible_architecture == "x86_64" or ansible_architecture == "amd64"
+
+  # - name: Copy JTDX from local file (because automated downloads don't work reliably) (AMD64)
+  #   copy:
+  #     src: "{{ playbook_dir }}/../files/jtdx-2.2.0-rc155_2004_amd64.deb"
+  #     dest: "/home/{{ ham_user }}/hamradio/jtdx-2.2.0-rc155_2004_amd64.deb"
+  #   when: ansible_architecture == "x86_64" or ansible_architecture == "amd64"
+
+  # - name: Install JTDX from local file (because automated downloads don't work reliably) (AMD64)
+  #   become: yes
+  #   apt:
+  #     deb: /home/{{ ham_user }}/hamradio/jtdx-2.2.0-rc155_2004_amd64.deb"
+  #   when: ansible_architecture == "x86_64" or ansible_architecture == "amd64"
+
+  # - name: Remove JTDX installer from local file (because automated downloads don't work reliably) (AMD64)
+  #   file:
+  #     path: /home/{{ ham_user }}/hamradio/jtdx-2.2.0-rc155_2004_amd64.deb"
+  #     state: absent
+  #   when: ansible_architecture == "x86_64" or ansible_architecture == "amd64"
+
+  - name: Install dependent libraries
+    become: yes
+    package:
+      name: "{{ item }}"
+      state: present
+    with_items:
+      - libqt5websockets5-dev
+      - libqt5serialport5-dev
+    retries: 5
+    delay: 30
+    register: result
+    until: result.failed == false
     when: ansible_architecture == "x86_64" or ansible_architecture == "amd64"
 
-  - name: Install JTDX from local file (because automated downloads don't work reliably) (AMD64)
-    become: yes
-    apt:
-      deb: /home/{{ ham_user }}/hamradio/{{ jtdx_installer }}"
+  - name: Git clone latest jtdx sources
+    git:
+      repo: https://www.github.com/jtdx-project/jtdx
+      dest: /home/{{ ham_user }}/hamradio/jtdx
+    retries: 5
+    delay: 30
+    register: result
+    until: result.failed == false
     when: ansible_architecture == "x86_64" or ansible_architecture == "amd64"
 
-  - name: Remove JTDX installer from local file (because automated downloads don't work reliably) (AMD64)
+  - name: Create directory jtdx/build
     file:
-      path: /home/{{ ham_user }}/hamradio/{{ jtdx_installer }}"
-      state: absent
+      path: /home/{{ ham_user }}/hamradio/jtdx/build
+      state: directory
+    when: ansible_architecture == "x86_64" or ansible_architecture == "amd64"
+      
+  - name: Build Dev jtdx CMakeFiles
+    command: cmake .. -DWSJT_SKIP_MANPAGES=ON
+    args:
+      chdir: /home/{{ ham_user }}/hamradio/jtdx/build
     when: ansible_architecture == "x86_64" or ansible_architecture == "amd64"
 
+  - name: Build jtdx
+    command: cmake --build . --target install
+    args:
+      chdir: /home/{{ ham_user }}/hamradio/jtdx/build
+    when: ansible_architecture == "x86_64" or ansible_architecture == "amd64"
 #
```

Comment:
@wsciaroni After building and installing, we need to clean up the sources/object files by removing them. Here's an example of what to add:

  - name: Remove JTDX build directory
    file:
      path: /home/{{ ham_user }}/hamradio/jtdx
      state: absent

Can you please add this?